spec/integration/discourse_slack/slack_spec.rb

```diff
@@ -219,6 +235,21 @@
             "filter" => "mute",
             "tags" => [tag.name]
           ])
+
+          post '/slack/command.json',
+            text: "status",
+            channel_name: "welcome",
+            token: token
+
+          json = JSON.parse(response.body)
+
+          text = I18n.t("slack.message.status.all_categories",
+                    channel: "#welcome",
```

Comment:
Style: The indentation is really weird here. It is indented 3 spaces from `I18n` in the line above while the other addition in the code uses 2 spaces. Anyway, I prefer to just indent two spaces from the line above. Indenting more doesn't help with code readability at all and is just extra effort.